pkg/documents/documents.go

```diff
@@ -246,26 +264,44 @@ func NewDocumentRepo(logger log.Logger, db *sql.DB) DocumentRepository {
 		logger: logger,
 	}
 }
+
+func (r *sqlDocumentRepository) findActiveDocument(customerID string, documentID string, namespace string) error {
+	query := `select documents.document_id from documents
+inner join customers on customers.namespace = ? where documents.customer_id = ? and documents.document_id = ? and documents.deleted_at is null
+limit 1;`
+	stmt, err := r.db.Prepare(query)
+	if err != nil {
+		return fmt.Errorf("error preparing query: %v", err)
+	}
+	defer stmt.Close()
+
+	var docID string
+	if err := stmt.QueryRow(namespace, customerID, documentID).Scan(&docID); err != nil {
+		return err
+	}
+	return nil
+}
+
 func (r *sqlDocumentRepository) getCustomerDocuments(customerID string, namespace string) ([]*client.Document, error) {
 	query := `select document_id, documents.type, content_type, uploaded_at from documents join customers on customers.
 	namespace = ? where documents.customer_id = ? and documents.deleted_at is null`
 	stmt, err := r.db.Prepare(query)
 	if err != nil {
-		return nil, fmt.Errorf("getCustomerDocuments: prepare %v", err)
+		return nil, fmt.Errorf("error getting customer documents - prepare %v", err)
```

Comment:
One more small nitpick:
Avoid putting "error" or "failed" in error messages. The reasoning is outlined here: https://github.com/uber-go/guide/blob/master/style.md#error-wrapping